docs/user/security/authentication/index.asciidoc

```diff
@@ -21,32 +49,33 @@ To successfully log in to {kib}, basic authentication requires a username and pa
 
 The session cookies that are issued by the basic authentication provider are stateless. Therefore, logging out of {kib} when using the basic authentication provider clears the session cookies from the browser, but does not invalidate the session cookie for reuse.
 
+NOTE: You can configure only one Basic provider per {kib} instance.
+
 For more information about basic authentication and built-in users, see
 {ref}/setting-up-authentication.html[User authentication].
 
 [[token-authentication]]
 ==== Token authentication
 
-Token authentication allows users to login using the same {kib} provided login form as basic authentication, and is based on the Native security realm or LDAP security realm that is provided by {es}. The token authentication provider is built on {es} token APIs. The bearer tokens returned by {es}'s {ref}/security-api-get-token.html[get token API] can be used directly with {kib} using the `Authorization` request header with the `Bearer` scheme.
+Token authentication allows users to log in using the same {kib} provided login form as basic authentication, and is based on the Native security realm or LDAP security realm that is provided by {es}. The token authentication provider is built on {es} token APIs.
```

Comment:
unrelated to your changes, but since we're here, we might as well add AD to the list, since it's technically supported by our basic auth provider.

```suggestion
Token authentication allows users to log in using the same {kib} provided login form as basic authentication, and is based on the Native, LDAP, or Active Directory security realm that is provided by {es}. The token authentication provider is built on {es} token APIs.
```